projects/plugins/jetpack/extensions/blocks/recipe/step/edit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+function RecipeStepEdit( { className } ) {
+	const RECIPE_TEMPLATE = [
+		[
+			'core/heading',
+			{
+				level: 5,
+				className: 'wp-block-jetpack-recipe-step-name',
+			},
+		],
+		[
+			'core/paragraph',
+			{
+				className: 'wp-block-jetpack-recipe-step-desc',
+			},
+		],
+		[
+			'core/image',
+			{
+				className: 'wp-block-jetpack-recipe-step-image',
+			},
```

Comment:
I wasn't sure if the template was helpful, or a bit heavy handed. I'll remove the block template altogether here and just limit what can be placed within. This may negate the above issue regarding the heading level, as users will be able to choose whichever heading level they like.